Source/WebKit/UIProcess/WebPageProxy.h

```diff
@@ -42,6 +42,10 @@
 #include <WebCore/PlatformTextAlternatives.h>
 #endif
 
+#if PLATFORM(COCOA)
+#include "TextIndicatorStyle.h"
+#endif
```

Comment:
We should not be importing new headers in this file unless absolutely necessary. I think we can forward declare this enum?